nucliadb_protos/resources.proto

```diff
@@ -270,6 +262,22 @@ message FieldMetadata {
     repeated Relations relations = 13;
 }
 
+message QuestionAnswer {
+  string question = 1;
+  string question_language = 2;
+  string answer = 3;
+  string answer_language = 4;
+  repeated int64 paragraph_ids = 5;
```

Comment:
This should be stirng isn't it ? the paragraphs ids are `$RID/$FIELD/($SPLIT)/$START-$END`